.github/workflows/publish-release.yml

```diff
@@ -0,0 +1,38 @@
+name: Publish Release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: '0'
+    - uses: actions/setup-python@v2
+      with:
+        python-version: '3.8'
```

Comment:
I noticed the other repo is testing with 3.9. Should we be testing using the same versions and potentially multiple python versions like 3.6, 3.7, 3.8, and 3.9?

Reply:
We are testing with those versions.  This is the python version used for building and publishing the artifact.  3.8 works.  I switched the other repo to use the same Python for consistency, though eventually I think it needs to have tests that mock access and can run in all versions.  I just wasn't ready overhaul the test suite to get releases set up.